src/Protocol/ClientEncoder.php

```diff
@@ -35,4 +39,12 @@ public function __doRequest($request, $location, $action, $version, $one_way = 0
         // do not actually block here, just pretend we're done...
         return '';
     }
+
+    public function requestTarget($requestTarget)
```

Comment:
How does this method name relate to the one in the `Client`? It's my understanding that they should use related names?

Reply:
Oops. Will fix this.